index.html

```diff
@@ -44,129 +50,172 @@ <h1>I'm Abigael <br> Glad to see <br> you</h1>
-            </div>
-            
-            <div class="language-list">
-                <div class="language">
-                    <img src="images/js-icon.svg" alt="" aria-hidden="true">
-                    <p>JavaSript</p>
-                </div>
-                <div class="language">
-                    <img src="images/html-icon.svg" alt="" aria-hidden="true">
-                    <p>HTML</p>
+    <div class="about-me-section">
+        <div class="about">
+            <section class="about-me">
+                <h2>About <br> Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have 
+                    a project you need coded, don’t hestiate to contact me.</p>
+                <h3>Lets Connect</h3>
+                <img src="images/social-media-icons.png" alt="" aria-hidden="true">
+                <button type="submit">Get my resume</button>
+            </section>
+        </div>
+    
+        <!-- Languages Section -->
+        <div class="languages-section">
+            <section class="languages">
+                <div class="languages-title">
+                    <h4>Languages</h4>
+                    <img src="images/downarrow.svg" alt="">
                 </div>
-                <div class="language">
-                    <img src="images/css-icon.svg" alt="" aria-hidden="true">
-                    <p>CSS</p>
+                
+                <div class="language-list">
+                    <div class="language">
+                        <img src="images/js-icon.svg" alt="" aria-hidden="true">
+                        <p>JavaSript</p>
+                    </div>
+                    <div class="language">
+                        <img src="images/html-icon.svg" alt="" aria-hidden="true">
+                        <p>HTML</p>
+                    </div>
+                    <div class="language">
+                        <img src="images/css-icon.svg" alt="" aria-hidden="true">
+                        <p>CSS</p>
+                    </div>
                 </div>
-            </div>
-        </section>
- 
-        <section class="frameworks">
-            <h4>Frameworks</h4>
-            <img src="images/next.png" alt="" aria-hidden="true">
-        </section>
-
-        <section class="skills-a">
-            <h4>Skills</h4>
-            <img src="images/next.png" alt="" aria-hidden="true">
-        </section> 
+            </section>
+     
+            <section class="frameworks">
+                <h4>Frameworks</h4>
+                <img src="images/next.png" alt="" aria-hidden="true">
+            </section>
+    
+            <section class="skills-a">
+                <h4>Skills</h4>
+                <img src="images/next.png" alt="" aria-hidden="true">
+            </section> 
+        </div>
     </div>
+    
 
     <!-- Contact me form  -->
     <div class="contact-me-form">
```

Comment:
- [ ] Kindly adjust the about me section and contact form section to  take the full width at 768px, at 1024px and 1440px everything looks good, this willl make your project look better at all screensizes 👍🏿 

![Screenshot_106](https://user-images.githubusercontent.com/69638013/166307216-81e956b5-2c20-47aa-93a6-21d793ae3e4b.png)

Reply:
This has been resolved.